utils/post.js

```diff
@@ -12,6 +12,7 @@ const {
   Sequelize
 } = require('../databases/models');
 const {NO_POLL_OPTION_UUID, POST_TYPE_LINK, POST_VERB_POLL} = require('../helpers/constants');
+const deleteActivityFromUserFeed = require('../services/getstream/deleteActivityFromUserFeed');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `deleteActivityFromUserFeed` is added. However, it's important to ensure that the function is properly exported from the `../services/getstream/deleteActivityFromUserFeed` module. If not, this could lead to a `TypeError: deleteActivityFromUserFeed is not a function` error.

```diff
+ const deleteActivityFromUserFeed = require('../services/getstream/deleteActivityFromUserFeed');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->